src/transformers/models/code_llama/tokenization_code_llama.py

```diff
@@ -65,6 +65,7 @@ class CodeLlamaTokenizer(PreTrainedTokenizer):
     there is no padding token in the original model.
 
     Args:
+    The default configuration match that of the model []() which supports prompt infilling.
```

Comment:
Is this really expected to be here?

Reply:
It is, updated the tokenizer path! Nice catch

Reply:
(should be above too)